### PR TITLE
prow: Add more automatisation for k/kubectl

### PR DESCRIPTION
**Pull request by apelisse**
I have no idea what the `trigger` plugin does, but the name looks cool.

Fixes: https://github.com/kubernetes/kubectl/issues/122

### CODE REVIEW

**Comment by apelisse** [comment]
Also, how does tide knows if the tests are passing?

**Comment by apelisse** [comment]
cc @droot

**Comment by cjwagner** [comment]
> Also, how does tide know if the tests are passing?

It won't merge anything unless all the reported github statuses are green.

File: prow/plugins.yaml [hunk 1]
```diff
@@ -67,6 +67,10 @@ plugins:
   kubernetes/kops:
   - trigger
 
+  kubernetes/kubectl:
+  - trigger
```
Comment (cjwagner) [inline prow/plugins.yaml]: It doesn't look like there are any jobs configured for kubernetes/kubectl in `prow/config.yaml` so trigger won't have any jobs to trigger for this repo.

Comment (apelisse) [reply prow/plugins.yaml]: Cool, I'll just remove. We use travis for now, I'll revisit that when we/if we migrate.

**Comment by apelisse** [comment]
> It won't merge anything unless all the reported github statuses are green.

Perfect! I'll update.

**Pull request merged**